spec/features/01_user_sign_in_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+feature "sees sign in options" do
+  scenario "see all sign in options" do
+    visit user_index_path
+    expect(page).to have_content('Sign In')
+    expect(page).to have_content('Sign Out')
+  end
+
+  scenario "user successfully signs in" do
+    user = FactoryGirl.create(:email)
+    user_password = FactoryGirl.create(:password)
+
+    visit user_sign_in_path
+    click_on 'Sign In'
+    fill_in 'Email', :with => user
+    fill_in 'Password', :with => user_password
+
+    expect(page).to have_content(user.email)
+  end
+
+  scenario "user successfully signs out" do
+    user = FactoryGirl.create(:email)
+    user_password = FactoryGirl.create(:password)
+
+    visit user_sign_in_path
+    click_on 'Sign In'
+    fill_in 'Email', :with => user
+    fill_in 'Password', :with => user_password
```

Comment:
You should have an authentication helper, so you can just write a line of code like `sign_in_as user` which will sign the person in for you. Here is an example https://github.com/davidrf/adventure_planner/blob/master/spec/support/helpers/authentication_helper.rb